assets/js/components/PostSearcherAutoSuggest.js

```diff
@@ -59,6 +59,7 @@ export default function PostSearcherAutoSuggest( {
 	onClose = () => {},
 	placeholder = '',
 } ) {
+	const lastfetchRequest = useRef();
```

Comment:
Given "last" and "fetch" are different words here, let's make this:

```suggestion
	const lastFetchRequest = useRef();
```

(it'll needed to be updated below as well)

Reply:
Oops, thanks! 😄